plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar310/python.jjt

```diff
@@ -1167,7 +1158,6 @@ void closed_pattern() : {}
     | attr() [class_pattern()]
     | sequence_pattern()
     | mapping_pattern()
-    | wildcard_pattern()
```

Comment:
It may be nice to leave this line commented and add a note saying that the `_` is actually matched as a regular `Name` (and then acted upon in the TreeBuilder to set up the context properly).